q2_diversity_lib/plugin_setup.py

```diff
@@ -243,3 +250,100 @@
         citations['lozupone2011unifrac'],
         citations['mcdonald2018unifrac']]
 )
+
+# ------------------------ Dispatch ------------------------
+plugin.methods.register_function(
+    function=q2_diversity_lib.alpha_passthrough,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={'metric': Str % Choices(alpha.METRICS['NONPHYLO']['UNIMPL'])},
+    outputs=[('vector', SampleData[AlphaDiversity])],
+    input_descriptions={'table': "The feature table containing the samples "
+                        "for which a selected metric should be computed."},
+    parameter_descriptions={'metric':
+                            'The alpha diversity metric to be computed.'},
+    output_descriptions={'vector': "Vector containing per-sample values "
+                                   "for the chosen metric."},
+    name="Alpha Passthrough (non-phylogenetic)",
+    description="Computes a vector of values for each samples in a feature "
+                "table using the scikit-bio implementation of a chosen alpha "
+                "diversity metric."
+)
+
+
+plugin.methods.register_function(
+    function=q2_diversity_lib.beta_passthrough,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={'metric': Str % Choices(beta.METRICS['NONPHYLO']['UNIMPL']),
+                'pseudocount': Int % Range(1, None),
+                'n_jobs': Int % Range(1, None) | Str % Choices(['auto'])},
+    outputs=[('distance_matrix', DistanceMatrix)],
+    input_descriptions={
+        'table': ('The feature table containing the samples over which beta '
+                  'diversity should be computed.')
```

Comment:
No need for the superfluous parenthesis.

```suggestion
        'table': 'The feature table containing the samples over which beta '
                 'diversity should be computed.'
```

I think in general you have gone with the paren-free style in most of the other actions defined in here, so please scour and remove all unnecessary parens.